MVP/BusinessLogic/ScoresModel.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ScoresRepository.swift
+//  MVP
+//
+//  Created by matt bowlin on 10/4/19.
+//  Copyright © 2019 Insight. All rights reserved.
+//
+
+//import Foundation
+
+struct Score: Codable {
+    let id, playerAID: Int
+    let playerA: PlayerA
```

Comment:
I know this was auto generated from the website, but I would rename the PlayerA object to Player